### ISSUE TITLE
When using BPagination, `Failed to execute whenDefined on CustomElementRegistry' can be seen on console.

### ISSUE THREAD

**Issue by yuto168** [bootstrap-vue-next/bootstrap-vue-next#2151]
### Describe the bug

As stated in the subject, Failed to execute 'whenDefined on 'CustomElementRegistry' on console when using BPagination.
I have confirmed this on stackblitz as well as on my own project.

![image](https://github.com/user-attachments/assets/2efa9a93-6fc7-4d6d-b2ec-aa193b05facb)

### Reproduction

https://stackblitz.com/edit/nuxt-starter-bootstrap-vue-next?file=app.vue

### Used Package Manager

npm

### Validations

- [X] Have tested with the latest version. This is still alpha version and sometime things change rapidly.
- [X] Follow our [Code of Conduct](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by loicduong** [comment]
I don't think this is a bug. The issue arises from the chorme extension. As seen in the image, the id eimadpbcbfnmbkopoojfekhnkhdbieeh belongs to the extension [Dark Reader](https://chromewebstore.google.com/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh). You can disable the extension or use incognito mode to check.
